### ISSUE TITLE
Feature request: Nonce edit

### ISSUE THREAD

**Issue by tobutridaz** [poanetwork/nifty-wallet#396]
Feature request: A settings toggle that would add a nonce field for all transactions, allowing to override the default nonce. 

Scenario: On Ethereum network, when a txn is underpriced with gas, instead of speeding it, the transaction might become redundant at certain time, so a replacing it with a completely different transaction is useful, but requires to override nonce for the new transaction.

**Comment by vbaranov** [comment]
@tobutridaz it is already possible to set custom nonce for any transaction. It has been implemented in this PR https://github.com/poanetwork/nifty-wallet/pull/379. And this feature is available in the currently published Nifty wallet 5.1.1. Isn't this you are looking for?

**Comment by tobutridaz** [comment]
I'm running 5.1.1 and below is a transaction popup after interacting with a dapp. Where do I edit the nonce? 
![brave_2020-07-08_10-45-28](https://user-images.githubusercontent.com/12191364/86898037-79779280-c108-11ea-9bf6-dd9176e79274.png)

**Comment by vbaranov** [comment]
Oh yes. I see it now. The nonce's editing is available in a previous screen when someone sends tx from the wallet itself. And there is no way to get to that previous screen if transaction originated from dApp. I should open access to that previous screen from the confirmation screen.

**Comment by tobutridaz** [comment]
Yes please, or just add a nonce field on the confirm tx window as well.
